app/talk/search-result.jsx

```diff
@@ -67,7 +70,7 @@ export default class TalkSearchResult extends React.Component {
         )}
         <CommentContextIcon comment={comment} />
         <CommentLink comment={comment} project={this.props.project}>{comment.discussion_title}</CommentLink>
-        <Markdown content={comment.body} project={this.props.project} />
```

Comment:
The way to debug this bug was to add `debug` here, which tells `Markdown` to show error messages in the page:
```jsx
<Markdown debug content={comment.body} project={this.props.project} />
```